libnavui-androidauto/build.gradle

```diff
@@ -41,15 +41,16 @@ dependencies {
     // This defines the minimum version of Maps, Navigation, and Search which are included in
     // this SDK. To upgrade the SDK versions, you can specify a newer version in your downstream
     // build.gradle.
-    api("com.mapbox.navigation:android:2.5.0-rc.1")
+//    api("com.mapbox.navigation:android:2.5.0-rc.1")
+    api project(':libnavigation-android')
```

Comment:
@kmadsen Quick question. Why are we referecing the published navigation SDK artifact instead of module like this?

Reply:
This would compile against whatever is on the top of the current branch in Nav SDK, and that would be used for a release of Android Auto artifact (resulting in undefined POM, or dependency on a snapshot). Refs https://github.com/mapbox/mapbox-navigation-android/pull/5773#discussion_r865960135.

If AA and Nav SDK were following the same release cycle and versioning it wouldn't be a problem, but since they aren't we need to have an explicit mechanism to determine what AA is building against.

Reply:
I see. That makes things slightly complicated. Meaning I cannot touch **libnavui-androidauto** module in this PR and I have to wait until all extracted voice module interfaces are available in the next `com.mapbox.navigation:android` artifact.

Reply:
> That makes things slightly complicated.

Yeah sorry this was a known trade-off. For what it's worth, this is simpler than it used to be because when integrating with 1tap we have to use snapshots for development. Here we can use `project(':libnavigation-android')` 😄

Reply:
Actually, you may want to test this with a snapshot anyways. Wondering if there will be visibility issues

For example, 1tap references `interface MapboxAudioGuidance` but it is in an internal package.

Reply:
Oh never mind it's using the extension. https://github.com/mapbox/1tap-android/blob/f38e99e448709a72d5b037b2bb32d6aedb374dcc/map/src/main/java/com/mapbox/onetap/map/presentation/ui/MapFragment.kt#L388.

Reply:
Moved **libnavui-androidauto** changes to a separate PR https://github.com/mapbox/mapbox-navigation-android/pull/5846